holoviews/plotting/bokeh/renderer.py

```diff
@@ -82,7 +82,10 @@ def __call__(self, obj, fmt=None, doc=None):
             if bokeh_version < '0.12.6':
                 raise RuntimeError('Bokeh png export only supported by versions >=0.12.6.')
             from bokeh.io import _get_screenshot_as_png
-            img = _get_screenshot_as_png(plot.state)
+            if bokeh_version > '0.12.6':
+                img = _get_screenshot_as_png(plot.state, None)
+            else:
+                img = _get_screenshot_as_png(plot.state)
```

Comment:
Yes, should have mentioned it, it's my fault. I was going to bring it up but we needed something that would work for this dev release and I only noticed last night.